tasks/ActivityShikigami/assets.py

```
@@ -93,10 +93,13 @@ class ActivityShikigamiAssets:
 	# Ocr Rule Assets
 	# 挑战 
 	O_FIRE = RuleOcr(roi=(1124,599,96,50), area=(1123,570,106,100), mode="Single", method="Default", keyword="挑战", name="fire")
+	# 20260725 缩小体力和活动体力的检测范围，范围太大，有可能把0识别为10的情况。
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)